mitodl/ocw-hugo-themes
# Display no internet connection warning only in offline YouTube videos

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/2734

### Description (What does it do?)
This PR removes `Viewing videos requires an internet connection` warning from online theme, making it appear only for YouTube videos in offline theme under no (or very low) internet connection.

### Screenshots (if appropriate):
<img width="1094" alt="image" src="https://github.com/user-attachments/assets/a17e93a5-dff7-4c46-8624-c5db2cd435b8">


### How can this be tested?
First, checkout to this branch.

1. `yarn build ~/path/ocw-content-rc/<any course that has YouTube videos>/ ~/path/ocw-hugo-projects/ocw-course-v2/config-offline.yaml`
    * You can use `ibrahims-cat-course`
    * It has a menu on the left named videos
    * Get a fresh clone of it using `yarn start course ibrahims-cat-course`
2. Go to dist directory of course, and open `index.html`
3. Turn off your internet connection
4. Go to videos / lecture videos
5. Open any video and it should display the internet connection warning.

In addition, you can test that archive videos do not show the warning. (Prior to this PR, they showed warning).
1. Download the course content for `18.06-spring-2010` (`yarn start course 18.06-spring-2010`)
2. Source `ocw-studio` production AWS credentials and set them up in a terminal
3. In the same terminal change directory to the course content repo you downloaded above
4. 
```
mkdir -p content/static_resources
aws s3 sync s3://ol-ocw-studio-app-production/courses/18-06-linear-algebra-spring-2010/ ./content/static_resources/
```
5. Once the static resources have been downloaded, run the following in the `ocw-hugo-themes` directory: `yarn build ~/path/ocw-content-rc/18.06-spring-2010/ ~/path/ocw-hugo-projects/ocw-course-v2/config-offline.yaml`
6. In your file browser, go to your course content repo and enter the dist folder, then double click on `index.html` to open the offline site in a web browser
7. Turn off your internet connection
8. From `index.html`, go to Video Lectures from left navbar, open any Lecture Video
9. It should not display warning.
